app/Http/Controllers/WebhookController.php

```diff
@@ -98,4 +98,22 @@ public function destroy(Webhook $webhook)
     {
         //
     }
+
+    public function enable($id)
+    {
+        $webhook = Auth::user()->webhooks->find($id);
+        $webhook->status = WebhookStatus::ENABLED;
+        $webhook->save();
+        return 'This webhook successfully enabled';
+    }
+
+    public function disable($id)
+    {
+        $webhook = Auth::user()->webhooks->find($id);
+        $webhook->status = WebhookStatus::DISABLED;
+        $webhook->save();
+        return 'This webhook successfully disabled';
+    }
 }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.TooMany</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 117</br>Expected 1 blank line at end of file; 3 found